.github/workflows/typescript-publish.yml

```
@@ -50,25 +58,81 @@
+            
+            
+            ### NPM Package
+            📦 [View on NPM](https://www.npmjs.com/package/brainyflow/v/${{ steps.version.outputs.new_version }})
+          draft: false
+          prerelease: false
+
+      - name: Notify Discord
+        if: steps.version.outputs.has_changes == 'true' && env.DISCORD_WEBHOOK_URL != ''
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+        run: |
```

actionlint (1.7.7)

property "changelog" is not defined in object type {changelog_text: {conclusion: string; outcome: string; outputs: {string => string}}; current_version: {conclusion: string; outcome: string; outputs: {string => string}}; version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)